src/onnx/parse_deconvolution.cpp

```diff
@@ -95,6 +95,8 @@ struct parse_deconvolution : op_parser<parse_deconvolution>
             check_attr_sizes(
                 kdims, values["dilation"].size(), "PARSE_CONV_TRANSPOSE: inconsistent dilations");
         }
+
+        // TODO: nothing is done with this?
```

Comment:
what's the todo item here ?

Reply:
The auto_pad option doesn't have any functionality for parse_deconvolution. It doesn't look like it's implemented. I will update the TODO comment to say this.